fighters/chrom/src/acmd/smashes.rs

```diff
@@ -10,16 +10,21 @@ unsafe fn chrom_attack_s4_s_game(fighter: &mut L2CAgentBase) {
     if is_excute(fighter) {
         WorkModule::on_flag(boma, *FIGHTER_STATUS_ATTACK_FLAG_START_SMASH_HOLD);
     }
-    frame(lua_state, 12.0);
+    frame(lua_state, 10.0);
+    if is_excute(fighter) {
+        FT_MOTION_RATE(fighter, 3.0/(13.2-10.0));
+    }
+    frame(lua_state, 13.2);
```

Comment:
I'm curious, why 13.2 specifically

Reply:
Matching Roy fsmash data made by Black Calculus. Presumably the animation frame was chosen to allow for better hitbox interpolation.